common/templates/context.go

```diff
@@ -418,6 +418,9 @@ func (c *Context) newContextFrame(cs *dstate.ChannelState) *ContextFrame {
 func (c *Context) ExecuteAndSendWithErrors(source string, channelID int64) error {
 	out, err := c.Execute(source)
 
+	// trim whitespace for accurate character count
+	out = strings.TrimSpace(out)
```

Comment:
This would need changes on the UI with char counter.

Reply:
Which changes? It doesn't impact the custom command editor at all, or the length of the code, it's only the output of executed commands before it's sent

Reply:
Oh alright, I assumed it was removing the whitespace before saving a command. My bad.